init-jjs-root.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+mkdir -p "$1"
+mkdir "$1/"{bin,etc,lib,tmp,var}
+cat > "$1/etc/jjs.toml" << EOF
+toolchain-root= "/opt/jjs-tc/root"
+EOF
+mkdir "$1/etc/toolchains"
+cat > "$1/etc/toolchains/cpp.toml" << EOF
+name="cpp"
+suffix="cpp"
+[[build]]
+argv=["/usr/bin/g++", "\$(System.SourceFilePath)", "-o", "\$(System.BinaryFilePath)", "-std=c++17", "-Wall", "-Wextra", "-Wpedantic", "-DJJS"]
+[run]
+argv=["\$(System.BinaryFilePath)"]
+EOF
```

Comment:
Why is it better thar init-jjs-root tool?

Reply:
That piece of shit doesnt work for some reason. Also I think shell scripts are more appropriate for this task.

Reply:
But anyway, config shouldn't be hardcoded.
What *should* be done:
```
if has_python()
    soft --with=python
    cp example-config/toolchains/python.toml JJS_ROOT/etc/toolchains/python.toml
if has_gcc()
    soft --with=gcc
    cp example-config/toolchains/gcc.toml JJS_ROOT/etc/toolchains/gcc.toml
```
and so on.
I'm now working on soft to teach in how to install `python` and `gcc`, but if you want you can rewrite it